model/event/subscriptions.go

```diff
@@ -238,14 +248,26 @@ func (s *Subscription) String() string {
 	return out
 }
 
-func FindSubscriptionsByOwner(owner string) ([]Subscription, error) {
+func FindSubscriptionsByOwner(owner string, ownerType OwnerType) ([]Subscription, error) {
 	if len(owner) == 0 {
 		return nil, nil
 	}
 	query := db.Query(bson.M{
-		subscriptionOwnerKey: owner,
+		subscriptionOwnerKey:     owner,
+		subscriptionOwnerTypeKey: ownerType,
 	})
 	subscriptions := []Subscription{}
 	err := db.FindAllQ(SubscriptionsCollection, query, &subscriptions)
 	return subscriptions, errors.Wrapf(err, "error retrieving subscriptions for owner %s", owner)
 }
+
+func IsValidOwnerType(in string) bool {
```

Comment:
I'm almost included to move this into the rest package

Reply:
I feel like this makes more sense in the model. It's not related to rest routes at all - it's just validation that's needed since go doesn't support enums